.omeroci/lib-test

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -eux
+
+docker run --link $CID:omero --net $NETWORK --rm -d -p 9449:9449 --name omero-prometheus-tools test -s omero -u root -w omero -c /opt/omero-prometheus-tools/etc/prometheus-omero-counts.yml -v
```

Comment:
As a follow on to this PR, it'd be good to hear suggestions on making `lib-test` extensible. E.g. would it have sufficed if the script:

```
docker ... $LIB_TEST_EXTRA_DOCKER_ARGS ... test "$@"
```

Reply:
That would work for a self-contained docker test, but here I'm running addition test commands below.

Reply:
You'd use it as a `super(class).__init__()` style method, basically reducing whatever copy-n-pasting is possible.